docs/modeltranslation/registration.rst

```diff
@@ -127,6 +127,40 @@ the translated field ``image`` to the ``NewsWithImage`` model.
     inheritance` broke the project (if you had always subclassed
     ``TranslationOptions`` only, there is no risk).
 
+.. versionadded:: 0.19.0
+
+If you need to specify the languages available for each model, you can do so using the ``allowed_languages``
+argument::
+
+    from modeltranslation.translator import translator, TranslationOptions
+    from news.models import News, NewsWithImage
+
+    class NewsTranslationOptions(TranslationOptions):
+        fields = ('title', 'text',)
+
+    class NewsWithImageTranslationOptions(TranslationOptions):
+        allowed_languages=('en','uk')
```

Comment:
Why are you using 'allowed_languages' instead of just languages?

I'd prefer `languages`, or meybe `translation_languages`, to better highlight the intention.